daemon/emer-aggregate-tally.c

```diff
@@ -240,16 +255,26 @@ emer_aggregate_tally_constructed (GObject *object)
                      ")");
 }
 
+static gboolean
+close_db (sqlite3  *db,
+          GError  **error)
+{
+  if (db == NULL)
+    return TRUE;
+
+  return CHECK (sqlite3_close (db));
+
```

Comment:
Stray newline.

Reply:
I'm trying to squash this newline away in the commit where it was added, but it is proving surprisingly fiddly. I thought I was quite good at rebase!

Reply:
Killed it, I think!

Reply:
yep, github is attaching this to some other newline now.